CSharpMath.Tests/PreTypesetting/MathListBuilderTest.cs

```diff
@@ -973,16 +973,16 @@ public class MathListBuilderTest {
 
       Assert.Single(list);
       var latex = MathListBuilder.MathListToString(list);
-      Assert.Equal(RemoveWhiteSpacesFromString(latex), input);
+      Assert.Equal(input, RemoveWhiteSpacesFromString(latex));
     }
 
     [Fact]
-    public void TestListToString() {
+    public void TestListToString_int_a_to_b() {
       TestLatexToListAndBack(@"\int_a^b");
     }
 
     [Fact]
-    public void TestListToString1() {
+    public void TestListToString_int_wdf() {

```

Comment:
Same here. `wdf` has no actual meaning as it is just a combination of variables. Better give a name like `TestListToString_Integrals`.